homeassistant/components/azure_event_hub/strings.json

```diff
@@ -0,0 +1,50 @@
+{
+    "config": {
+      "step": {
+        "user": {
+          "title": "Setup your Azure Event Hub integration",
+          "data": {
+            "event_hub_instance_name": "Event Hub Instance Name",
+            "use_connection_string": "Use Connection String"
+          }
+        },
+        "conn_string": {
+          "title": "Connection String method",
+          "description": "Please enter the connection string for: {event_hub_instance_name}",
+          "data": {
+            "event_hub_connection_string": "Event Hub Connection String"
+          }
+        },
+        "sas": {
+          "title": "SAS Credentials method",
+          "description": "Please enter the SAS (shared access signature) credentials for: {event_hub_instance_name}",
+          "data": {
+            "event_hub_namespace": "Event Hub Namespace",
+            "event_hub_sas_policy": "Event Hub SAS Policy",
+            "event_hub_sas_key": "Event Hub SAS Key"
+          }
+        }
+      },
+      "error": {
+        "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+        "unknown": "[%key:common::config_flow::error::unknown%]"
+      },
+      "abort": {
+        "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
+        "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+        "cannot_connect": "Connecting with the credentails from the configuration.yaml failed, please remove from yaml and use the config flow.",
+        "unknown": "Connecting with the credentails from the configuration.yaml failed with an unknown error, please remove from yaml and use the config flow."
+      }
+    },
+    "options": {
+      "step": {
+        "options": {
+          "title": "Options for the Azure Event Hub.",
+          "data": {
+            "send_interval": "Interval for sending batches to the hub.",
+            "max_delay": "Max delay before discarding a message."
```

Comment:
Why is it important to be able to customize these options?

Reply:
these options determine the behaviour, the first around how often it sends the latest events, longer interval means less messages with larger payloads and more latency before the message arriving. The max delay is about discarding messages that are too old, so if for some reason HA is struggling to catch up, after a while messages are just discarded instead of keep trying to send them, so a security measure to make sure this integration doesn't block everything else, at some point it will just discard these old messages which frees up the rest of the system. And they were already in the yaml config, but in most cases a sensible default is used, so hence using the options for that.

Reply:
Ok, but why do we need to customize them? Isn't a sane default enough?

Reply:
for the second one I would be ok with that, the interval has implications for costs vs latency (on the azure side) so having that customizable is important.

Reply:
Then I suggest we remove max_delay option in the options flow. We can still import the option from YAML for existing setups if we want to ease migration.

Reply:
sounds good

Reply:
I've pretty radically deleted the max_delay so let me know!